omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_postgres_validator.rb

```diff
@@ -190,7 +190,7 @@ def backend_verify_postgres_version(connection)
     # Note that we're looking for the same major, and using our minor as the minimum version
     # This provides compatibility with external databases that use < 9.6 before we make use
     # of any features available in > 9.2.
-    unless (major == REQUIRED_MAJOR) && (minor >= REQUIRED_MINOR)
+    unless major == 12 || major == 13 || ((major == REQUIRED_MAJOR) && (minor >= REQUIRED_MINOR))
```

Comment:
any reason for hardcoding this?

Reply:
I changed it to constants (I assume that's what you want?).
```
unless SUPPORTED_MAJOR.include?(major) || ((major == REQUIRED_MAJOR) && (minor >= REQUIRED_MINOR))
```